components/src/core/channel-value/channel-value.component.ts

```diff
@@ -19,5 +21,5 @@ export class ChannelValueComponent {
     @Input() units: string;
     @Input() fontSize = 'inherit';
     @Input() prefix = false;
-    @Input() color = 'inherit';
+    @Input() color;
```

Comment:
do we need this at all if we're removing 'style' attributes and relying on SCSS?

Reply:
Huayun is working on edits that will remove this input.  I also think fontSize should go away, along with all respective stories/knobs.